.gitignore

```diff
@@ -1,6 +1,8 @@
 # build output
 dist
 .next
+bs
+.merlin
```

Comment:
Can there be in the gitignore of the example? or in the gitignore of the examples folder 👌